### ISSUE TITLE
build server is not capable of handling git submodules and will ignore them

### ISSUE THREAD

**Issue by alberttwong** [resin-os/resinos#215]
I wanted to just post this since it's not posted anywhere else.  I understand that currently the build server is not capable of handling git submodules and will ignore them.  Unfortunately, I do use submodules for my git project.

**Comment by willnewton** [comment]
This sounds like it might be a backend issue. @CameronDiver ?

**Comment by agherzan** [comment]
It is definitely and I hit it last week too. Had to hack it in the Dockerfile.

**Comment by CameronDiver** [comment]
You're correct, via the git server the builder only gets the context without the submodules. I'll look at getting this added in. Thanks for bringing it to our attention @alberttwong :+1:
